### ISSUE TITLE
Coturn Reload Config without down

### ISSUE THREAD

**Issue by kholisrag** [coturn/coturn#878]
Hi, is there away to reload the config without kill / terminate existing coturn daemonon / service,

so basically, we want to autoscale, using `alternate-server` options, we already can update the `alternate-server` config dynamicly,

but how to reload the config? without terminating existing client connection / terminating existing coturn service / daemon?
thanks!

**Comment by ggarber** [comment]
No, that's not supported right now.   If there are enough people interested it looks like a nice feature to be added, although it can be hard to make it work for all the configuration options so probably it needs to be done for specific ones.

**Comment by eakraly** [comment]
It is common to use SIGHUP to reload configuration but in coturn SIGHUP is used to reset logfile
I would suggest to reload configuration file on a different signal (SIGUSR1 for example) which will give you what you want

And @ggarber comment above is super important - it is not clear what/if options will be reloaded at runtime and how they will impact the server. For example, can you change SSL certificate at runtime? Can you change number of threads? Change runtime listening device?

If anything, we can start with a single configuration option that will be reloaded - and that will be `alternate-server` that you need.

As usual - PR is welcome @kholisrag !

**Comment by jdfalls** [comment]
This would be a very helpful feature. I'm running coturn in docker, and have certbot automated to renew letsencrypt certs for turn server, but don't have a way to auto-reload coturn after the cert renewal (outside of using a chron job on the host I guess).

When I was running coturn as a regular app, not in docker, this was straightforward -- I just used a certbot renewal-hook which copied the certs and restarted coturn. But now using docker, certbot (which is running in it's own container) does not have the access/permission to restart coturn.

(nginx has a "-s reload" flag [uses SIGHUP] that allows this to be scripted in a docker compose file, which works nicely in conjunction with certbot. Would be great if coturn could do the same.) 

Thanks!

**Comment by eakraly** [comment]
@jdfalls can you use SIGUSR2 instead for now to reload certificates?
Until we all figure out what to do next with the signals and reloading of configuration
